maat-court-data-api/src/main/java/gov/uk/courtdata/enums/LoggingData.java

```diff
@@ -28,4 +28,10 @@ public void putInMDC(String value) {
       MDC.put(key, value);
     }
   }
+
+  public void removeFromMdc(String key) {
+    if (Objects.nonNull(key)) {
```

Comment:
Might be worth checking if not blank (empty string too) - not sure if we're allowed them in a map.